pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Tags'
+weight: 63
+---
+
+--- Unique Description about AsyncAPI document to be inserted here ---
+
+A `tag` is a label or category that helps you logically group related objects such as channels or servers within an event-driven system. The `tags` object provide a structured way to group channels, operations or other components based on their functionality, purpose and/or any other relevant criteria.
+
+The `tags` object serves as a container for defining and managing tags in a systematic manner. Within the `tags` object, you can define individual tags, specifying a name and providing an optional description that adds further context to the tag's purpose or usage. 
+
+Once tags are defined within the `tags` object, they can be assigned to channels, operations or any other component using the tags property. By including the relevant tag names in the tags property, objects such as channels, servers or components are associated with the appropriate tags, grouping them based on their shared characteristics.
+
+Additionally, `tags` can also be included in the [`components` object](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#componentsObject) of an AsyncAPI document, which enables the reusability of the tags. `tags` included in the `components` object can be re-used by [reference objects](https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#referenceObject).
```

Comment:
remember to user relative links to docs, never absolute as the moment we do release and remove `v3` subdomain, they will be broken

example: instead of `https://v3.asyncapi.com/docs/reference/specification/v3.0.0-next-major-spec.12#componentsObject` do `/docs/reference/specification/v3.0.0-next-major-spec.12#componentsObject`